Examples/README.md

```diff
@@ -0,0 +1,17 @@
+# Examples
+
+These two examples projects illustrate how VelocidiSDK can be used in a project. ObjcExample is written in Objective-C and SwiftExample is written in Swift.
+
+## Instructions
```

Comment:
As discussed offline, can you improve this section?

Reply:
Check b3fbf26